### DISCUSSION TITLE
arduion libray issues

### DISCUSSION THREAD

**Discussion by lakshkunde** [bxparks/AceButton#113] [Q&A]
I am not able to install this library in my arduino

**Comment by marmilicious** [comment]
Can you provide more information? How are you trying to install it? Are you getting some sort of error?

**Comment by bxparks** [comment]
Thanks @marmilicious, for trying to answer a question with no details.
Also, moving to Discussions, as explained in https://github.com/bxparks/AceButton#FeedbackAndSupport

**Comment by bxparks** [comment]
No response after 2 weeks.
Closing this to remove clutter and noise.
